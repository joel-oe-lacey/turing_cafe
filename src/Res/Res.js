import React from 'react';
import './Res.scss';

const Res = ({id, name, date, time, guestCount}) => {
    return (
        <section key={id} className='reservation'>
            <h3>{name}</h3>
            <p>{date}</p>
            <p>`${time} PM`</p>
            <p>`Number of Guests: ${guestCount}`</p>
            <button>Cancel</button>
        </section>
    )
}

export default Res;