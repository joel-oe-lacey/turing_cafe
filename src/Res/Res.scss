.reservation {
    height: 40%;
    width: 18%;
    background-color: #FFFFFF;
    border: 4px solid teal;
    color: teal;
    border-radius: 5%;
    margin: 1%;

    button {
        height: 10%;
        width: 80%;
        background-color: teal;
        color: #FFFFFF;
    }
}