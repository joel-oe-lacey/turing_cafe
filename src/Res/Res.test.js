import React from 'react';
import { shallow } from 'enzyme';
import Res from './Res';

describe('Res', () => {
    const mockReservation = {
        id:'1',
        name:'Arthur Dent',
        date:'8/13',
        time:'8',
        number:'42'
    }

    it('Should match snapshot', () => {
        const wrapper = shallow(<Res {...mockReservation} />);
        expect(wrapper).toMatchSnapshot()
    })
})