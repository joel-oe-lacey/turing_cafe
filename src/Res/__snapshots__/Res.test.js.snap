// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Res Should match snapshot 1`] = `
<section
  className="reservation"
  key="1"
>
  <h3>
    Arthur Dent
  </h3>
  <p>
    8/13
  </p>
  <p>
    8 PM
  </p>
  <p>
    Number of Guests: 42
  </p>
  <button>
    Cancel
  </button>
</section>
`;
